src/Model/EditorReducer.re

```diff
@@ -0,0 +1,108 @@
+open Oni_Core;
```

Comment:
This was extracted from `Editor` because it depends on `Actions`, but should be moved back once it has its own `msg` type, along with a bunch of other reducers and stores/connectors.